discretize/TensorMesh.py

```diff
@@ -160,6 +160,24 @@ def gridN(self):
         return self._getTensorGrid('N')
 
     @property
+    def gridH(self):
+        """Cell widths in grid."""
+        H = self.h
+        if self.dim == 1:
+            return H
+        if self.dim == 2:
+            H = np.meshgrid(H[0],H[1])
```

Comment:
could you please install a pep8 linter? (see for example: http://docs.simpeg.xyz/content/api_core/api_practices.html#style). This really helps with readability and maintenance in the long run.

eg. there should be a space after the comma
```
H = np.meshgrid(H[0], H[1])
```